mgmt/rest/v2/mock/mock_config_manager.go

```diff
@@ -70,19 +70,25 @@ func (MockConfigManager) DeletePluginConfigDataNodeFieldAll(fields ...string) cd
 // rest_v2_test.go on the plugin config routes found in mgmt/rest/server.go
 const (
 	SET_PLUGIN_CONFIG_ITEM = `{
-  "user": "Jane"
+  "config": {
```

Comment:
@jcooklin, the response change is from word `body` to `config`. if that's your concern, I can change it to `body`. In this context, `config` makes more sense. Please try it. With API V1, after set config, the response is:

**V1 API response after updating a config item:**
```
{
  "meta": {
    "code": 200,
    "message": "Plugin config item(s) set",
    "type": "config_plugin_item_created",
    "version": 1
  },
  "body": {
    "leader": "thunder",
    "password": "p@ssw0rd",
    "path": "whatever",
    "user": "whoever"
  }
}
``` 

**V2 API response after updating a config item:**
```
{
"config": {
"leader": "thunder",
"password": "p@ssw0rd",
"path": "whatever",
"user": "whoever"
}
}
```